ramls/accounts.raml

```diff
@@ -11,6 +11,7 @@ types:
   account: !include accountdata.json
   accountdataCollection: !include accountdataCollection.json
   errors: !include raml-util/schemas/errors.schema
+  patronNotice: !include patronNotice.json
```

Comment:
Minor version here and in module-descriptor should be increased because of this change